bin/ContributeRecipe

```diff
@@ -218,7 +218,10 @@ then
    git stash pop
    git branch -D "submit-$program-$version"
 else
+   git config --global hub.protocol https
    hub --version &> /dev/null || Die "You need to install 'hub' to auto-submit pull-requests"
-   hub pull-request -b "$compileUpstreamBranch" -h "submit-$program-$version" -m "$program $version"
+   hub fork --remote-name origin-fork
+   git push --set-upstream  origin-fork "submit-$program-$version"
+   hub pull-request -b "origin/$compileUpstreamBranch" -h "origin-fork/submit-$program-$version" -m "$program $version"
```

Comment:
This does not work directly either. But we can instead make a clickable link to create PR via web page. Or find a way to make this work from CLI: